packages/gatsby/src/redux/actions.js

```diff
@@ -411,7 +478,7 @@ actions.createParentChildLink = ({ parent, child }, plugin) => {
  */
 actions.createPageDependency = ({ path, nodeId, connection }, plugin = ``) => {
```

Comment:
Just overloading it for now... it'd be a breaking change :-(

I'll add a note to change it in v2.